lib/internal/bootstrap_node.js

```diff
@@ -672,10 +672,28 @@
   };
 
   NativeModule.wrapper = [
-    '(function (exports, require, module, internalBinding, process) {',
+    '(function (exports, require, module, internalBinding, process, { ' +
+      'CHECK, CHECK_EQ, CHECK_GE, CHECK_GT, CHECK_LE, CHECK_LT, CHECK_NE }) {',
     '\n});'
   ];
 
+  const CHECK = (expr) => {
+    if (expr)
+      return;
+    const e = new Error(`${expr} == true`);
```

Comment:
this will just be a boolean for all the CHECK_* macros that use this, and won't be very descriptive, any ideas on how to fix that?